### PR TITLE
[NL] Add some Dutch sentences for climate

### PR DESCRIPTION
**Pull request by berendhaan**
Just some simple few sentences to start with. The naming conventions weren't very clear to me, may need some work.

### CODE REVIEW

**Comment by TheFes** [comment]
Hi Berend,
Besides the sentences themselves, you also need to add tests for the sentences.
It was a bit of a puzzle for me as well, but I managed to get my first PR merged earlier today. You can have a look at the files changed in my PR to see what needs to be added/changed

See: https://github.com/home-assistant/intents/pull/64#event-8097226710

**Comment by berendhaan** [comment]
> Hi Berend,
> 
> Besides the sentences themselves, you also need to add tests for the sentences.
> 
> It was a bit of a puzzle for me as well, but I managed to get my first PR merged earlier today. You can have a look at the files changed in my PR to see what needs to be added/changed
> 
> 
> 
> See: https://github.com/home-assistant/intents/pull/64

Hi Fes,
Thanks for your mention 😊 will look into it!

File: sentences/nl/climate_HassClimateGetTemperature.yaml [hunk 1]
```diff
@@ -2,4 +2,6 @@ language: nl
 intents:
   HassClimateGetTemperature:
     data:
-    - sentences: []
+    - sentences:
+        - "<hoe_warm> is het"
```
Comment (balloob) [inline sentences/nl/climate_HassClimateGetTemperature.yaml:6]: `<hoe_warm>` is geen bestaande expansion rule

**Comment by synesthesiam** [comment]
Looks like we just need some more test sentences for HassClimateGetTemperature

**Comment by berendhaan** [comment]
> Looks like we just need some more test sentences for HassClimateGetTemperature

Incoming :)

**Review by balloob — APPROVED** [review]

**Pull request merged**